### ISSUE TITLE
Problem with start-storybook

### ISSUE THREAD

**Issue by bryanjtc** [eirslett/storybook-builder-vite#252]
I did a clean install on a project using node 16. I deleted the package-lock and node-modules. Running the project throws this error. Not sure if the issue is with the storybook vite builder or storybook. I'm using storybook v6.4.19. The project ran without errors on node 14.

![image](https://user-images.githubusercontent.com/49354825/154830257-768c293e-660a-4a5a-84d1-d1addfd7a400.png)
![image](https://user-images.githubusercontent.com/49354825/154830264-33f7a5fa-2edd-450e-beb2-6b68e22bd3f0.png)
![image](https://user-images.githubusercontent.com/49354825/154830275-709d033b-c530-4568-b490-f2708c815dbd.png)

**Comment by IanVS** [comment]
Can you please share your `.storybook/main.js` configuration?  A repo that reproduces the problem would be even better.

**Comment by bryanjtc** [comment]
![image](https://user-images.githubusercontent.com/49354825/154849538-db7ae4e6-73e8-4880-8781-913e45dc99b4.png)

**Comment by IanVS** [comment]
From what I can tell, this must be a bug from the webpack storybook manager.  Do you still get the error if you don't use the vite builder?  What version of storybook are you using?

**Comment by IanVS** [comment]
Oh, I see 6.4.19.  very odd.

**Comment by IanVS** [comment]
Storybook does occasionally have windows bugs.  I'd suggest opening an issue in the main storybook repo, or searching to see if one already exists.

**Comment by bryanjtc** [comment]
I can confirm the error it's only with the vite builder. I just tried with the webpack5 builder and it works.
```
//const svgrPlugin = require('vite-plugin-svgr');

module.exports = {
  core: {
    builder: 'webpack5',
  },
  stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
  addons: [
    '@storybook/addon-links',
    '@storybook/addon-essentials',
    '@storybook/preset-scss',
    '@storybook/addon-a11y',
    'storybook-addon-performance/register',
  ],
  features: {
    storyStoreV7: true,
  },
  framework: '@storybook/react',
  /*async viteFinal(config, { configType }) {
    config.plugins = [svgrPlugin(), ...config.plugins];
    configType === 'DEVELOPMENT' &&
      (config.optimizeDeps.include = [
        ...(config.optimizeDeps?.include ?? []),
        'storybook-addon-performance',
        'd3-selection',
        'd3-scale',
        'd3-axis',
        '@storybook/client-api',
      ]);
    return config;
  },*/
  webpackFinal: async (config, { configType }) => {
    const fileLoaderRule = config.module.rules.find((rule) => rule.test && rule.test.test('.svg'));
    fileLoaderRule.exclude = /\.svg$/;
    config.module.rules.push({
      enforce: 'pre',
      test: /\.svg$/,
      use: [
        {
          loader: '@svgr/webpack',
          options: {
            exportType: 'named',
          },
        },
      ],
    });
    return config;
  },
};
```

**Comment by bryanjtc** [comment]
When using build-storybook it also throws an error with the vite builder and not the webpack5 builder.
![image](https://user-images.githubusercontent.com/49354825/154856424-fedaeab6-f6b3-4596-9790-cd88cf03bb7c.png)

**Comment by bryanjtc** [comment]
I decided to test with the latest prerelease version, 6.5.0-alpha.40, and it works. Both start-storybook and build-storybook. I guess the issue with webpack was solved in one of the prerelease versions of 6.5.
